libvast/test/default_table_slice.cpp

```diff
@@ -66,14 +82,14 @@ struct fixture {
   std::vector<value> subset(size_t from, size_t num) {
     return {test_values.begin() + from, test_values.begin() + (from + num)};
   }
-};
 
-template <class T>
-T unbox(caf::optional<T> x) {
-  if (!x)
-    FAIL("unable to unbox value");
-  return std::move(*x);
-}
+  template <class T>
+  T unbox(caf::optional<T> x) {
+    if (!x)
+      FAIL("unable to unbox value");
+    return std::move(*x);
+  }
+};
```

Comment:
Looks `unbox` is [now available](https://github.com/actor-framework/actor-framework/commit/848e5527f33ebe25e3e450d5d477b09bca888f97) to all CAF users.

I assume you just kept it in here because we already have a separate story to rip out now obsolete duplicate implementations.

Reply:
Almost. The commit in a currently [unmerged PR](https://github.com/actor-framework/actor-framework/pull/730). Once that hits `master` we can indeed rip out all `unbox` functions in VAST.